wkcuber/tests/test_cli.py

```diff
@@ -1,715 +1,165 @@
-import json
-import subprocess
-import sys
-from os import environ
+"""This module containes tests for the wkcuber CLI."""
+
+import os
+from contextlib import contextmanager
 from pathlib import Path
-from typing import Union
+from shutil import copytree
+from tempfile import TemporaryDirectory
+from typing import Iterator
 
-import numpy as np
 import pytest
-from upath import UPath
-from webknossos import DataFormat, Dataset, Mag
-from webknossos.dataset.dataset import PROPERTIES_FILE_NAME
-from webknossos.utils import copytree, rmtree
-
-from .constants import TESTDATA_DIR
-import subprocess
-import shlex
-
-
-def check_call(*args: Union[str, int, Path]) -> None:
-    try:
-        subprocess.check_call([str(a) for a in args])
-    except subprocess.CalledProcessError as e:
-        print(f"Process failed with exit code {e.returncode}: `{args}`")
-        raise e
-
-
-def count_wkw_files(mag_path: Path) -> int:
-    return len(list(mag_path.glob("**/x*.wkw")))
-
-
-MINIO_ROOT_USER = "TtnuieannGt2rGuie2t8Tt7urarg5nauedRndrur"
-MINIO_ROOT_PASSWORD = "ANTN35UAENTS5UIAEATD"
-MINIO_PORT = "8000"
-
-
-@pytest.fixture(scope="module")
-def remote_testoutput_path() -> UPath:
```

Comment:
Please add a test for the Zarr support. This fixture sets up a local S3 server that you can use as target.

Reply:
zarr still works after parsing with upaths